.github/workflows/ipv6_debug.yaml

```diff
@@ -0,0 +1,41 @@
+# ipv6 causes several tests to fail on ubuntu-latest;
+# see https://github.com/dask/distributed/issues/4514
+# This workflow demonstrates the issue.
+name: "Debug ipv6 issue #4514"
+
+on: []
+# on: [pull_request]  # Uncomment to enable
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.8"]
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup Conda Environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: latest
+          python-version: ${{ matrix.python-version }}
+          environment-file: continuous_integration/environment-${{ matrix.python-version }}.yaml
+          activate-environment: dask-distributed
+          auto-activate-base: false
+
+      - name: Install
+        shell: bash -l {0}
+        run: python -m pip install --no-deps -e .
+
+      - name: Test
+        shell: bash -l {0}
+        run: pytest distributed/cli/tests/test_dask_scheduler.py::test_defaults
+
+      - name: Debug with tmate on failure
+        if: ${{ failure() }}
+        uses: mxschmitt/action-tmate@v3
```

Comment:
Unsure if this file should go to master of if I should remove it here and paste it into #4514